src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-return  */
+import React, { useRef } from "react";
+import Omit from "lodash.omit";
+import { AutoField } from "uniforms-patternfly/dist/es6";
+import { AutoForm, ValidatedQuickForm, context } from "uniforms";
+import { createValidator } from "@app/Processor/ProcessorEdit/ConfigurationForm/validator";
+import { CustomJsonSchemaBridge } from "@app/Processor/ProcessorEdit/ConfigurationForm/CustomJsonSchemaBridge";
+import { useTranslation } from "react-i18next";
+
+interface ConfigurationFormProps {
+  configuration?: object;
+  onChange: (model: any) => void;
+  registerValidation: (validationFunction: () => boolean) => void;
+  schema: object;
+  readOnly?: boolean;
+}
+
+const ConfigurationForm = (props: ConfigurationFormProps): JSX.Element => {
+  const {
+    configuration = {},
+    onChange,
+    registerValidation,
+    schema,
+    readOnly = false,
+  } = props;
+  const { t } = useTranslation(["openbridgeTempDictionary"]);
+
+  const schemaValidator = createValidator(schema);
+  const bridge = new CustomJsonSchemaBridge(
+    schema,
+    schemaValidator,
+    t,
+    false,
+    false
+  );
+
+  // excluding error_handler, processors or steps
+  // @TODO remove it after https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/834 is merged
+  const properties = Omit(bridge.schema.properties, [
+    "error_handler",
+    "processors",
+    "steps",
+  ]);
+
+  const newRef = useRef<any>();
+
+  const validate = (): boolean => {
+    newRef.current?.submit();
+    const errors = schemaValidator(configuration);
+    return errors === null;
+  };
+
+  registerValidation(validate);
+
+  return (
+    <DynamicForm
+      validate={"onChangeAfterSubmit"}
+      schema={bridge}
+      model={configuration}
+      onChangeModel={onChange}
+      ref={(ref: any): void => (newRef.current = ref)}
+      disabled={readOnly}
+    >
+      {Object.keys(properties as { [key: string]: unknown }).map((key) => (
+        <AutoField key={key} name={key} disabled={readOnly} />
```

Comment:
Not sure if/how can we solve this problem. I guess we could try to pass an `ouiaId` prop to `AutoField` but I'm not sure all types of inputs will receive it. It depends mainly on `uniforms-patternfly`.

I think processor test is broken also because actions/sources have different names now. Their properties names have changed too.

Reply:
@jomarko https://issues.redhat.com/browse/MGDOBR-858